libs/libc/tls/task_getinfo.c

```diff
@@ -48,15 +49,7 @@
 
 FAR struct task_info_s *task_get_info(void)
 {
-  FAR struct task_info_s *info = NULL;
-  struct stackinfo_s stackinfo;
-  int ret;
+  FAR struct tls_info_s *info = up_tls_info();
 
-  ret = nxsched_get_stackinfo(-1, &stackinfo);
```

Comment:
remove the special handing of -1 from nxsched_get_stackinfo

Reply:
Should this change in another patch?

Reply:
It's more clean why we remove -1 support in the same patch.